locations/spiders/essar_oil_gb.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, Fuel, apply_category, apply_yes_no
+from locations.google_url import extract_google_position
+from locations.items import Feature
+
+
+# TODO: Address data not consistent between pages
+class EssarOilGBSpider(SitemapSpider):
+    name = "essar_oil_gb"
+    item_attributes = {"brand": "Essar", "brand_wikidata": "Q5399372"}
```

Comment:
https://github.com/search?q=repo%3Aalltheplaces%2Falltheplaces+Q5399372&type=code

But also probably just the US branch

Reply:
Non issue